0.17/files/scenario.sh

```diff
@@ -1,44 +1,42 @@
 #!/bin/sh -x
-if [ -z "$1" ]
-  then
-    echo "No argument supplied"
+if [ -z "$1" ]; then
+  echo "No argument supplied"
 fi
-SCENARIO=$1
+SCENARIO="$1"
 
 set -e
```

Comment:
If these are all changed to `set -euo pipefail`, then the issue of nonexistent variables being used (like the $SCENARIOS variable that was fixed in this PR) will count as an error going forward. (The pipefail part just turns on error checking for commands in pipe statements. Not directly relevant here but I think it's a good habit to do in bash scripts, and could help with a problem in the future.)